Makefile

```diff
@@ -65,6 +63,14 @@ DEBUG_BUILD ?= no
 # The latter is painfully slow on Mac OS X with Docker Desktop, so we default to using a
 # standalone volume in that case, and to bind mounting otherwise.
 UNAME_S := $(shell uname -s)
+UNAME_M := $(shell uname -m)
+
+BUILD_IMAGE := quay.io/stackrox-io/apollo-ci:$(shell sed 's/\s*\#.*//' BUILD_IMAGE_VERSION)
+ifeq ($(UNAME_S),Darwin)
+ifeq ($(UNAME_M),arm64)
+	BUILD_IMAGE = quay.io/rhacs-eng/sandbox:apollo-ci-stackrox-build-0.3.44-arm64
```

Comment:
Can we eventually derive this from `$BUILD_IMAGE`, once this is built in the same pipeline as the `apollo-ci` image? (Obviously yes - add a TODO?)

Reply:
Yes, will add a TODO. This image isn't exactly buildable in that pipeline because of the ARM support so it'll be a hack for now